tests/e2e/cypress/cypress/integration/examples/example-test-request.spec.js

```diff
@@ -81,46 +83,52 @@ describe('Template: librarian request', function() {
     cy.server();
     cy.route({method: 'DELETE', url: '/api/items/*'}).as('deleteItem');
     // Go to item detail view
-    cy.goToProfessionalDocumentDetailView(this.itemBarcode);
+    cy.goToProfessionalDocumentDetailView(documentPid);
     cy.get('#item-' + this.itemBarcode + ' div a[name=barcode]').click();
     // Remove request
     cy.get('#' + this.users.patrons.james.barcode + ' div [name=cancel]').click();
     cy.get('#modal-confirm-button').click();
-    // Go back to document detail view and remove item
-    cy.goToProfessionalDocumentDetailView(this.itemBarcode);
-    cy.get('#item-' + this.itemBarcode + ' [name=buttons] > [name=delete]').click();
-    cy.get('#modal-confirm-button').click();
-    cy.wait('@deleteItem');
+    // Go back to document detail view
+    cy.goToProfessionalDocumentDetailView(documentPid);
+    // Remove item
+    cy.apiDeleteResources('items', 'pid:"'+ itemPid + '"');
     // Remove document
-    cy.reload(); // Bug: need to reload the page to enable the remove button
-    cy.deleteRecordFromDetailView();
+    cy.apiDeleteResources('documents', 'pid:"'+ documentPid + '"');
     cy.logout();
   });
 
   // First test
   it('First test: a librarian makes a request for a patron', function() {
     console.log('first test');
-    cy.route('/api/item/*/can_request?library_pid=' + this.users.librarians.spock.libraryPid + '&patron_barcode=' + this.users.patrons.james.barcode).as('getCanRequest');
+    cy.route('/api/item/*/can_request?library_pid='
+      + this.users.librarians.spock.libraryPid
+      + '&patron_barcode='
+      + this.users.patrons.james.barcode)
```

Comment:
cy.route(\`/api/item/*/can_request?
library_pid=${this.users.librarians.spock.libraryPid}
&patron_barcode=${this.users.patrons.james.barcode}\`)